scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -3,16 +3,14 @@
 
 ---
 cmake_target_version: "3.13.1"
-clang_target_version: "7.1.0"
+clang_target_version: "8.0.0"
 shellcheck_target_version: "0.4.6"
 cmake_prefix: "/usr/local"
 
-#TODO: Flip to Clang-8 after https://github.com/openenclave/openenclave/pull/3574 is merged
-clang_binary_name: "clang-7"
+clang_binary_name: "clang-8"
 
 apt_packages:
   - "curl"
-  - "clang-7"
   - "clang-format-7"
```

Comment:
Can get rid of the format dependency :).

Reply:
I was under the impression that we would be using `clang-format-7`, unless @anakrish already made the format updates on his `clang-8` PR?

Reply:
using `clang-format-7` to format `clang-8` code sounds odd to me, but maybe I am missing some context.

Reply:
Let us stick to clang-format-7 for now. I'd like to avoid unnecessary churn at this point.

Reply:
Sure sounds good.